examples/poc/manifests/gateway/gateway.yaml

```diff
@@ -0,0 +1,23 @@
+
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: <GATEWAY-NAME>
+spec:
+  gatewayClassName: <GATEWAY-NAME>
+  listeners:
+    - name: http
+      protocol: HTTP
+      port: 8080
+    - name: llm-gw
+      protocol: HTTP
+      port: 8081
+---
```

Comment:
As @arkodg pointed out, a more EG aligned approach could be
* creating an HTTPRoute for the llm-gw port
* targeting the EnvoyExtensionPolicy to the HTTPRoute 
* using EnvoyPatchPolicy to patch the route to route the original destination cluter.


Dummy backend for the LLM HTTPRoute
```yaml
    apiVersion: gateway.envoyproxy.io/v1alpha1
    kind: Backend
    metadata:
      name: backend-dummy
    spec:
      endpoints:
        - fqdn:
            hostname: 'foo.bar.com'
            port: 8080

```

HTTPRoute for inference traffic

```yaml
    apiVersion: gateway.networking.k8s.io/v1
    kind: HTTPRoute
    metadata:
      name: llm-route
    spec:
      parentRefs:
        - name: gateway-btls
          sectionName: llm-gw
      rules:
        - backendRefs:
            - group: gateway.envoyproxy.io
              kind: Backend
 ```

EnvoyExtensionPolicy
```yaml
apiVersion: gateway.envoyproxy.io/v1alpha1
kind: EnvoyExtensionPolicy
metadata:
  name: ext-proc-policy
  namespace: default
spec:
  extProc:
    - backendRefs:
      - group: ""
        kind: Service
        name: instance-gateway-ext-proc
        port: 9002
      processingMode:
        request:
          body: Buffered
        response:
      messageTimeout: 5s
  targetRef:
    group: gateway.networking.k8s.io
    kind: HTTPRoute
    name: llm-route
```

Reply:
This is awesome. And basically exactly what we want for the endstate of LLM-I-GW. But, hopefully, supported at the GW level so patching & backends don't need to be enabled on GWs.

Thanks! Done.